README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Abstract
 
-This specification defines an extension of OpenID Connect to allow presentation of claims in the form of W3C Verifiable Credentials as part of the protocol flow in addition to claims provided in the `id_token` and/or via Userinfo responses.
+This specification defines additional parameters `vp_jwt`, `vp_ldp`, `vc_jwt`, `vc_ldp` to allow presentation of claims in the form of W3C Verifiable Credentials objects as part of the OpenID Connect protocol flow.
```

Comment:
It does more than that as it includes syntax for requesting vcs and vps and will also contain processing rules. What's wrong with the existing text?

Reply:
I think syntax for requesting vcs and vps should be different from defining the claims. as iterated in the PR comment

Reply:
Sure. Requesting something is different from representing something. However, I intend to come up with at least one reasonable end 2 end solution. That's important for implementers and it is the basis for a stable and reasonable concept. Just discussion claims in isolation is not sufficient because those claims serve a purpose. There is not a single JWT claim so far defined in isolation. verified_claims, for example, is part of OpenID 4 Identity Assurance. It would never have gotten the maturity it has now without the context. Does this mean verified_claims cannot be used in other contexts? Of course not! We use it heavily in token introspection responses. 

I suggest to first of all discuss this topic and then propose potential changes to our draft in a separate PR. So far, we have discussed to change the representation of the VCs/VPs from vp_token and id token as VP to universal claims in ID token (and other artefacts). We agreed to continue this discussion and you took the task to make a PR as basis for further discussions. The current PRs goes way beyond what we have discussed. It especially proposes removal of text that I consider important and have put significant work in. I don't agree with those changes.

Reply:
> The current PRs goes way beyond what we have discussed. It especially proposes removal of text that I consider important and have put significant work in. I don't agree with those changes.

yes, I understand. It went wrong when doing this PR. Let me revert